docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -895,7 +895,6 @@ The same as ‘today() - 1’.
 ## timeSlot {#timeslot}
 
 Rounds the time to the half hour.
-This function is specific to Yandex.Metrica, since half an hour is the minimum amount of time for breaking a session into two sessions if a tracking tag shows a single user’s consecutive pageviews that differ in time by strictly more than this amount. This means that tuples (the tag ID, user ID, and time slot) can be used to search for pageviews that are included in the corresponding session.
```

Comment:
Should leave this example, but describe it in a generic way, keeping the example of "sessions" in web analytics.